README.md

```diff
@@ -99,6 +99,7 @@ Currently documentation is available for the following clusters:
 * [UPPMAX-DEVEL](docs/uppmax-devel.md)
 * [UPPMAX](docs/uppmax.md)
 * [UZH](docs/uzh.md)
+* [PASTEUR](docs/pasteur.md)
```

Comment:
Thanks @rplanel . Would you mind adding this in alphabetical order please :smile:

Reply:
My bad, will fix this once home 😆